emukit/benchmarking/benchmark_result.py

```diff
@@ -0,0 +1,47 @@
+from typing import List
+
+import numpy as np
+
+
+class BenchmarkResult:
+    def __init__(self, loop_names: List[str], n_repeats: int, metric_names: List[str]):
+        """
+
+        :param loop_names: List of loop names
+        :param n_repeats: Number of random restarts in benchmarking
+        :param n_iterations: Number of iterations in each loop
+        :param metric_names: List of metric names
+        """
+
+        self.loop_names = loop_names
+        self.n_repeats = n_repeats
+        self.metric_names = metric_names
+
+        self._results = dict()
+        for loop_name in loop_names:
+            self._results[loop_name] = dict()
+            for metric_name in metric_names:
+                self._results[loop_name][metric_name] = []
+                for i in range(n_repeats):
+                    self._results[loop_name][metric_name].append([])
+
+    def add_results(self, loop_name: str, i_repeat: int, metric_name: str, metric_values: list) -> None:
```

Comment:
The fact that metric_values are not already a numpy array feels slightly weird. Is there any reason for that?